tests/solana.rs

```diff
@@ -63,6 +63,10 @@ struct AccountState {
     lamports: u64,
 }
 
+/// We have a special callback function which tests that during CPI, the correct
+/// parameters are passed in.
```

Comment:
```suggestion
/// We have a special callback function which tests that the correct
/// parameters are passed in during CPI.
```
Does this sound better?

Reply:
Yes, it does sound better